Tests/MapboxDirectionsTests/VisualInstructionTests.swift

```diff
@@ -29,16 +29,17 @@ class VisualInstructionsTests: XCTestCase {
             ],
             "secondary": nil,
             "view": [
-              "text": "CA01610_1_E",
-              "components": [
-                [
-                  "text": "CA01610_1_E",
-                  "type": "guidance-view",
-                  "imageURL": "https://www.mapbox.com/navigation"
+                "text": "CA01610_1_E",
+                "components": [
+                    [
+                        "text": "CA01610_1_E",
+                        "type": "guidance-view",
+                        "imageURL": "https://www.mapbox.com/navigation",
+                        "subType": "cityreal"
```

Comment:
For tests, it would be great to check the other subtypes and their encodings to verify they all work as expected.

Reply:
Tests for encoding and decoding were added.